core/domain/feedback_services.py

```diff
@@ -366,6 +366,30 @@ def _get_thread_from_model(thread_model):
         thread_model.created_on, thread_model.last_updated)
 
 
+def get_new_messages_for_exp_id(exp_id):
+    """Fetch the feedback threads for the given exploration, and use those to
+    finally get the new (open) feedback messages.
```

Comment:
Actually we use "open" and "new" interchangeably wrt to the dashboard. That's because what we display as New Feedback on the dashboard frontend is actually the value of open feedback. 
Standardizing the naming seems like a good idea, I'll do that.